chainer_chemistry/functions/activation/softmax.py

```diff
@@ -0,0 +1,12 @@
+from chainer import functions
+
+
+def softmax(x, axis=1, mask=None, mask_value=1e10):
+    if mask is None:
+        h = x
+    else:
+        if x.shape != mask.shape:
```

Comment:
a bit strict? But we can fix later.

Reply:
Sorry. I would like to suggest we can broadcast if the dimension(of input) is consistent. So please ignore this comment now.

Reply:
yeah may be we can broadcast internally but i am not sure about all use case yet.